lms/djangoapps/discussion/views.py

```diff
@@ -7,23 +7,20 @@
 import logging
 
 from django.contrib.auth.decorators import login_required
-from django.conf import settings
 from django.core.context_processors import csrf
 from django.core.urlresolvers import reverse
 from django.contrib.auth.models import User
 from django.http import Http404, HttpResponseBadRequest
-from django.utils.translation import ugettext_noop
+from django.shortcuts import render_to_response
```

Comment:
One point I forgot: the template being rendered just has to start with ``## mako`` to make this work.